unified-scheduler-pool/src/lib.rs

```diff
@@ -102,6 +100,32 @@ enum CheckPoint<'a> {
 type CountOrDefault = Option<usize>;
 type AtomicSchedulerId = AtomicU64;
 
+#[derive(Debug)]
+pub enum SupportedSchedulingMode {
+    Either(SchedulingMode),
+    Both,
+}
```

Comment:
This will be turned into 2 variants of `VerificationOnly/Full` if #6737 lands first.